src/clj/rems/api/services/command.clj

```diff
@@ -28,6 +34,11 @@
          (mapcat #(approver-bot/generate-commands %))
          doall)))
 
+(defn run-process-managers [new-events]
+  (concat
+   (run-entitlements new-events)
+   (run-approver-bot new-events)))
```

Comment:
The call to `rems.email.core/generate-emails!` could be moved from `rems.api.services.command/command!` into this function the same way as `run-entitlements`.

Reply:
heh just noticed the same